statsmodels/survey/survey_model.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+import statsmodels.api as sm
+import summary_stats as ss
+
+# need to put in init_args and fit_args. Then do error checking for
+# model_class, args, and y
+
+class SurveyModel(object):
+
+    def __init__(self, design, model_class, y, init_args=None, fit_args=None):
```

Comment:
I don't think you should have "y" as an argument.  The model_class already has its data, which will not change (only the weights will change).